generator/src/test/java/com/scottlogic/deg/generator/cucumber/Decimal.feature

```diff
@@ -0,0 +1,94 @@
+Feature: User can specify that a number is of type decimal and does not have any decimal places
+
+Background:
+  Given the generation strategy is full
+    And there is a field foo
+    And foo is of type "decimal"
+    And foo is anything but null
+
+### Numeric Constraints
+
+Scenario: Greater than constraint with decimal type produces valid decimals
+  Given foo is greater than 10
+    And the generator can generate at most 5 rows
+  Then the following data should be generated:
+    | foo                     |
+    | 10.00000000000000000001 |
+    | 10.00000000000000000002 |
+    | 10.00000000000000000003 |
+    | 10.00000000000000000004 |
+    | 10.00000000000000000005 |
+
+Scenario: Greater than or equal to constraint with decimal type produces valid decimals
+  Given foo is greater than or equal to 10
+    And the generator can generate at most 5 rows
+  Then the following data should be generated:
+    | foo                     |
+    | 10                      |
+    | 10.00000000000000000001 |
+    | 10.00000000000000000002 |
+    | 10.00000000000000000003 |
+    | 10.00000000000000000004 |
+
+#594 - Change to descending order?
```

Comment:
Yeah I think this should be ignored with the expected values in descending order